gapic-generator-java/DEVELOPMENT.md

```diff
@@ -73,6 +75,13 @@ bazelisk test //...  # integration tests
     bazelisk run //test/integration:update_redis
     ```
 
+- To update all integration tests you can use this command:
+
+  ```sh
+    # In the repository root directory
+    bazelisk run //test/integration:update_asset && bazelisk run //test/integration:update_credentials && bazelisk run //test/integration:update_iam && bazelisk run //test/integration:update_kms && bazelisk run //test/integration:update_pubsub && bazelisk run //test/integration:update_logging && bazelisk run //test/integration:update_redis && bazelisk run //test/integration:update_storage && bazelisk run //test/integration:update_library && bazelisk run //test/integration:update_compute && bazelisk run //test/integration:update_bigtable && bazelisk run //test/integration:update_apigeeconnect 
```

Comment:
Thanks for this :) I've always wished for something like `bazelisk run //test/integration:update_all` but wasn't able to get it working. This will do for now